Prace_domowe/Praca_domowa6/Grupa4/Yevhenii_Vinichenko/pd6.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p>Warto w tym miejscu dodać także do wykresu błąd oryginalnych zdjęć. W obecnej formie to porównanie jest utrudnione.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/392/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>